src/en/disabilities/index.html

```diff
@@ -0,0 +1,32 @@
+---
+title: Most Common Types of Disability
+layout: layouts/base.njk
+description: Here are some of the most commonly found types of disabilities in the workplace, and their related accessibility considerations.
+---
+
+<div class="row wb-eqht">
```

Comment:
FYI - There is now a equal height that only leverage CSS. See the working example here: https://wet-boew.github.io/wet-boew/demos/eqht-css/eqht-css-en.html